modules/nw-metallb-example-addresspool.adoc

```diff
@@ -38,17 +38,23 @@ metadata:
   namespace: metallb-system
 spec:
   addresses:
```

Comment:
For a single reserved IP address is the `addresses` parameter still correct to use?